pkg/cmd/admin/registry/registry.go

```diff
@@ -543,3 +568,25 @@ func generateSecretsConfig(
 
 	return secrets, volumes, mounts, extraEnv, len(defaultCrt) > 0, nil
 }
+
+func generateSecurityContext(conf *RegistryConfig) *kapi.PodSecurityContext {
+	result := &kapi.PodSecurityContext{}
+	if len(conf.SupplementalGroupRange) > 0 {
+		result.SupplementalGroups = []int64{}
+		for _, val := range conf.SupplementalGroupRange {
+			groupID, err := strconv.ParseInt(val, 10, 64)
+			if err != nil {
+				// Don't need to report the error here as invalid values will be denied in
+				// Complete().
+				continue
+			}
+			result.SupplementalGroups = append(result.SupplementalGroups, groupID)
```

Comment:
BTW to be consistent in the code below and reduce lines of code this can be re-written to:
```go
	if groupID, err := strconv.ParseInt(val, 10, 64); err == nil {
		result.SupplementalGroups = append(result.SupplementalGroups, groupID)
	}
```